src/vs/workbench/workbench.common.main.ts

```diff
@@ -322,8 +325,8 @@ import 'vs/workbench/contrib/feedback/browser/feedback.contribution';
 // User Data Sync
 import 'vs/workbench/contrib/userDataSync/browser/userDataSync.contribution';
 
-// Profiles
-import 'vs/workbench/contrib/profiles/common/profiles.contribution';
+// User Data Profiles
+import 'vs/workbench/contrib/userDataProfile/browser/userDataProfile.contribution';
```

Comment:
Should this be core though? If profiles end up being a core thing, I would add the UI pieces to core as well I think.